aten/src/ATen/native/mps/operations/MultiTensorApply.h

```diff
@@ -153,6 +157,54 @@ struct FusedSgdEncodingFunctor {
   }
 };
 
+template <>
+struct FusedSgdEncodingFunctor<false> {
+  void operator()(
+    id<MTLComputeCommandEncoder>& computeEncoder,
+      id<MTLBuffer>& tensorArgumentBuffer,
+      const MetadataArguments& metadata_arguments,
+      const double weight_decay,
+      const double lr,
+      const bool maximize
+    ) const {
+      float weight_decay_lv = weight_decay;
+      float lr_lv = lr;
+      uint8_t maximize_lv = maximize;
```

Comment:
curious, what does lv stand for here?

Reply:
Was thinking of lvalue. Probably there is a better way to name it.